.github/workflows/POAM.yml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Team Sync
+on:
+  schedule:
+    - cron: 0 * 1 1-12 *
+
+jobs:
+  create_issue:
+    name: Create monthly POAM issue
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Create monthly POAM issue
+        run: |
+          if [[ $CLOSE_PREVIOUS == true ]]; then
+            previous_issue_number=$(gh issue list \
+              --label "$LABELS" \
+              --json number \
+              --jq '.[0].number')
+            if [[ -n $previous_issue_number ]]; then
+              gh issue close "$previous_issue_number"
+              gh issue unpin "$previous_issue_number"
+            fi
+          fi
+          new_issue_url=$(gh issue create \
+            --title "$TITLE" \
+            --assignee "$ASSIGNEES" \
+            --label "$LABELS" \
+            --body "$BODY")
+          if [[ $PINNED == true ]]; then
+            gh issue pin "$new_issue_url"
+          fi
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_REPO: ${{ github.repository }}
+          TITLE: POAM - [MONTH] '24
+          ASSIGNEES: mahoneycm
```

Comment:
Can we add the active role as dev instead of an individual assignee?

Reply:
Are you aware of a better way to add multiple quoted `LABEL`'s in yaml?

We need to wrap the label in quotes since it includes a colon, but I could not add a comma-separated list of them here without breaking the syntax. I believe if there isn't a colon in the label, we can just add the values without the quotes and separate them with commas. For now I was experimenting with just adding `Type: Task` as the current label, but replaced that with `Role: Dev` for now! Removed the `ASSIGNEESS` field as well.

[a2745ae](https://github.com/uswds/uswds-team/pull/366/commits/a2745ae2dbfd8f916e462aecb51c9d1c9eb4b7d7)

Reply:
Wrapping in quotes is probably the best way. Is that not working?

Also, for multiple labels you could use previous actions for reference. For example below is an action for USWDS.

https://github.com/uswds/uswds/blob/369cf3fdf9abf721a357e864a0f6adbf667ba862/.github/workflows/add-issue-labels.yml#L2-L6

Reply:
I was concerned that wrapping them in quotes (i.e. `"Role: Dev, Type: Task"`) would cause them to be read as a single label. I went ahead and updated in [9e3434f](https://github.com/uswds/uswds-team/pull/366/commits/9e3434ffb3b538197ada13931db2f57819f88873) to give it a shot! Let